lib/cli.js

```diff
@@ -29,6 +29,11 @@ class PdfOptions {
         describe: 'enable watch mode',
         type: 'boolean'
       })
+      .addOption('preview', {
+        default: false,
+        describe: 'open the otherwise headless launched browser for debugging the converted document',
```

Comment:
I think we shouldn't use debugging:

```suggestion
        describe: 'open the otherwise headless browser for inspecting the generated HTML document (before it gets converted to PDF)',
```

I think it's important to say that the browser will open the generated HTML page. The "converted document" could suggest that we are referring to the final output (ie. the PDF document).

Reply:
well "before it gets converted to PDF" is wrong, right? :P. The generation will be aborted if you abort the preview. Otherwise it sounds fine to me.

Reply:
The idea was to describe the conversion process: AsciiDoc -> HTML -> PDF.
So, I meant: you are inspecting the exact same HTML document that will be used to generate the PDF (when you are not using the preview mode).

But it's confusing... and maybe not the right time to explain how Asciidoctor PDF works under the hood (in 3 words) :laughing:

Reply:
well I do not mind too much :)